bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/dialogs/SearchPattern.java

```diff
@@ -105,39 +139,48 @@ public class SearchPattern {
 
 	private int allowedRules;
 
+	private boolean substringSearch;
+
+	private boolean matchPrefix;
+
+	private boolean matchSuffix;
+
 	/**
 	 * Creates a new instance of SearchPattern with the following match rules
 	 * configured: {@link #RULE_EXACT_MATCH} | {@link #RULE_PREFIX_MATCH} |
 	 * {@link #RULE_PATTERN_MATCH} | {@link #RULE_CAMELCASE_MATCH} |
-	 * {@link #RULE_BLANK_MATCH} )
-	 *
+	 * {@link #RULE_BLANK_MATCH}.
```

Comment:
Documentation should mention this uses `DEFAULT_RULES`

Reply:
OK, done.